frontend/app-development/features/SettingsModalButton/SettingsModal/components/Tabs/AbouTab/AboutTab.module.css

```
@@ -1,3 +1,47 @@
 .contentWrapper {
   max-width: 500px;
 }
+
+.createdFor {
+  display: flex;
+  align-items: center;
+  background-color: var(--fds-semantic-surface-neutral-subtle);
+  border-radius: 4px;
+  padding-inline: 12px;
+  padding-block: 20px;
+  margin-top: 10px;
+}
+
+.createdBy {
+  display: flex;
+  align-items: center;
+  margin-block: 10px;
+  color: var(--fds-semantic-text-action-active);
+}
```

Ref. forrige kommentar, så kan vi flytte litt CSS hit og slette en CSS klasse. 😊 Foreslår at vi setter gap til 4px, så teksten får en nærmere relasjon til ikonet som tilhører. Hva tenker du? 
```suggestion
.createdBy {
  display: flex;
  gap: 4px;
  align-items: center;
  margin-block: 10px;
  color: var(--fds-semantic-text-action-active);
}
```